chain/client/src/client_actor.rs

```diff
@@ -80,6 +80,8 @@ pub struct ClientActor {
     /// Adversarial controls
     pub adv: crate::adversarial::Controls,
 
+    // Address of this ClientActor. Can be used to send messages to self.
+    address: Addr<ClientActor>,
```

Comment:
Perhaps a better name would be `me: Addr<CLientActor>`. "address" doesn't add to what's already state in the type, while "me" contians an extra bit of info. Also, it's shorter :D

Reply:
I changed to my_address. I agree that me would be nice, but Client has a field `me` that stores the account_id of this node. I thought it might be confusing to add another `me` in ClientActor